### PR TITLE
Migration of logs RBAC permissions

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?

- Moves the logs RBAC permissions section in the RBAC permissions docs into its own guide.
- Updates the partial so we link to the new guide url.
- Adds further reading link.

### Motivation
The RBAC permissions page is now automatically generated for multiple products, and were creating individual guides for each section that will be linked out to as content keeps growing.

### Preview
https://docs-staging.datadoghq.com/sarina/logs-rbac-reorg/logs/guide/
https://docs-staging.datadoghq.com/sarina/logs-rbac-reorg/logs/guide/logs-rbac-permissions
https://docs-staging.datadoghq.com/sarina/logs-rbac-reorg/account_management/rbac/permissions

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by sarina-dd** [comment]
@bgdeutsch Hey there! Was able to update the URL for the logs RBAC permissions docs and it looks like the preview is working fine. Can you review whenever you get a chance? Thanks in advance for helping us with this! 😸

File: content/en/logs/guide/logs-rbac-permissions.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+* If this user has scoped [Read Index Data](#logs-read-index-data) permission on `audit` and `errors` indexes, this user only sees `service:api` logs within these indexes.
+* If this user has [livetail](#logs-livetail) permission, this users sees only sees `service:api` logs in the livetail.
+
+
+### `logs_read_index_data`
+
+Grants a role read access on some number of log indexes. Can be set either globally or limited to a subset of log indexes.
+
+To scope this permission to a subset of indexes, first remove the `logs_read_index_data` and `logs_modify_indexes` permissions on the role. Then
```
Comment (kayayarai) [inline content/en/logs/guide/logs-rbac-permissions.md:318]: Should there be a colon after "Then", like it's introducing the tabs? or did this sentence get chopped off?

File: content/en/logs/guide/logs-rbac-permissions.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+[2]: /api/v2/roles/#list-permissions
+[3]: /api/v1/logs-indexes/#get-all-indexes
+{{% /tab %}}
+{{< /tabs >}}
+
+### `logs_live_tail`
+
+Grants a role the ability to use the [Live Tail][20] feature.
+
+This permission is global, and grants access to the livetail irregardless of [Log Read Index Data](#logs-read-index-data) permission.
```
Comment (kayayarai) [inline content/en/logs/guide/logs-rbac-permissions.md:361]:
```suggestion
This permission is global, and grants access to the livetail regardless of [Log Read Index Data](#logs-read-index-data) permission.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Review by bgdeutsch — COMMENTED** [review]
Looks good to me!   Just one thing, the legacy logs permissions `href` attributes are hard-coded [here](https://github.com/DataDog/documentation/blob/master/layouts/partials/rbac-permissions-table.html#L71-L78), do we want to update these to point to the new logs rbac guide page as well?

File: layouts/partials/rbac-permissions-table.html [hunk 3]
```diff
@@ -68,18 +68,18 @@ <h2 id="{{ anchorize $permission_group }}">{{ $permission_group }}</h2>
...
-          <a href="#logs_live_tail"><code>logs_live_tail</code></a>
+          <a href="/logs/guide/logs-rbac-permissions/#logs_live_tail"><code>logs_live_tail</code></a>
         </td>
         <td>Access the live tail feature</td>
         <td>false</td>
       </tr>
       <tr>
         <td>
-          <a href="#logs_live_tail"><code>logs_read_index_data</code></a>
+          <a href="/logs/guide/logs-rbac-permissions/#logs_live_tail"><code>logs_read_index_data</code></a>
```
Comment (sarina-dd) [inline layouts/partials/rbac-permissions-table.html:78]:
```suggestion
          <a href="/logs/guide/logs-rbac-permissions/#logs_read_index_data"><code>logs_read_index_data</code></a>
```

Comment (sarina-dd) [reply layouts/partials/rbac-permissions-table.html:78]: Looks like this was previously incorrect; updating so it redirects to the correct location!

**Comment by sarina-dd** [comment]
@kayayarai Pushed the final commit 😸  Once the build is finished, we are good to merge!

**Pull request merged**